pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -283,6 +283,7 @@ stdenv.mkDerivation rec {
     homepage = "http://www.qemu.org/";
     description = "A generic and open source machine emulator and virtualizer";
     license = licenses.gpl2Plus;
+    mainProgram = "qemu-system-${stdenv.hostPlatform.qemuArch}";
```

Comment:
Sorry, please go back to `qemu-kvm` since it always exist and it is smart enough to figure out if KVM is available or not.

Reply:
> Sorry, please go back to `qemu-kvm` since it always exist and it is smart enough to figure out if KVM is available or not.

Ok.

Reply:
Done.